api/v1alpha1/openapi.yaml

```
@@ -3432,11 +3432,88 @@ components:
         - metadata
         - items
       description: FleetList is a list of Fleets.
+
+    DisruptionAllowance:
+      type: object
+      properties:
+        groupBy:
+          type: array
+          items:
+            type: string
+          description: List of labels to perform grouping for the disruption allowance.
+        minAvailable:
+          type: integer
+          description: The maximum number of unavailable devices allowed during rollout.
+        maxUnavailable:
+          type: integer
+          description: The minimum number of required available devices during rollout.
+      description: DisruptionAllowance defines the level of allowed disruption when rollout is in progress.
+
+    Percentage:
+      type: string
+      format: '^(?:(?:[1-9][0-9]?)|100|0)%$'
```

optional: Slightly simpler 
```suggestion
      format: '^(100|[1-9]?[0-9])%$'
```

---

OK